knexfile.js

```diff
@@ -1,52 +1,43 @@
 module.exports = {
-
   development: {
-    client: 'pg',
-    // connection: {
-    //   host:     'localhost',
-    //   port:     5432,
-    //   user:     'neighborhoodchef',
-    //   password: 'J4mb4l4y4!',
-    //   database: 'neighborhoodchef_dev',
-    // },
-    connection: 'postgres://neighborhoodchef:J4mb4l4y4!@localhost:5432/neighborhoodchef_dev',
+    client: "pg",
+    connection: process.env.DATABASE_URL,
```

Comment:
Do you need to include DATABASE_URL in the .env.example file? I don't see that change in this PR...

BTW, I've added the same connection string as POSTGRES_CONNECTION_STRING in my latest PR changes... If they're the same purpose, we should probably agree on which variable name to use. I'm not opposed to using DATABASE_URL.

Reply:
I reverted things back to NODE_ENV in dbConfig and POSTGRES_CONNECTION_STRING in the knexfile. I was originally having issues getting knex to connect to postgres when running tests, which is why changed things. It seems to work now though.